inc/Integrations/SalesforceD2C/SalesforceD2CDataSource.php

```diff
@@ -19,15 +19,15 @@ protected static function get_service_config_schema(): array {
 			'client_id' => Types::string(),
 			'client_secret' => Types::string(),
 			'enable_blocks' => Types::nullable( Types::boolean() ),
-			'organization_id' => Types::string(),
-			'shortcode' => Types::string(),
+			'instance_url' => Types::string(),
+			'store_id' => Types::string(),
 		] );
 	}
 
 	protected static function map_service_config( array $service_config ): array {
 		return [
 			'display_name' => $service_config['display_name'],
-			'endpoint' => sprintf( 'https://%s.api.commercecloud.salesforce.com', $service_config['shortcode'] ),
+			'endpoint' => $service_config['instance_url'],
```

Comment:
This is an improvement, I think the full URL makes things easier.

Reply:
Yip, 100% agreed as I realized this url can change depending on if you have a custom domain, etc